src/bp/ui-admin/src/Pages/Server/Checklist/index.tsx

```diff
@@ -0,0 +1,282 @@
+import { ServerConfig } from 'common/typings'
+import _ from 'lodash'
+import React, { FC, useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import api from '~/api'
+import PageContainer from '~/App/PageContainer'
+
+import { fetchServerConfig } from '../../../reducers/server'
+
+import Item from './Item'
+
+const NOT_SET = 'Not set'
+
+const getDisplayValue = (val: any) => {
+  if (val === undefined || val === null) {
+    return NOT_SET
+  } else if (val === false || val === true) {
+    return val.toString()
+  } else {
+    return val.length ? val.toString() : NOT_SET
+  }
+}
+
+const isSet = (value: any): boolean => value !== NOT_SET
+
+const protocol = window.location.protocol.substr(0, window.location.protocol.length - 1)
+
+interface Props {
+  serverConfig: ServerConfig
+  serverConfigLoaded: boolean
+  fetchServerConfig: () => void
+}
+
+const Container = props => {
+  return (
+    <PageContainer
+      title="Production Checklist"
+      helpText={
+        <span>
+          This is a checklist of recommended settings when running Botpress in production.
+          <br /> Environment variables are displayed in <Tag>gray</Tag> and values from the botpress.config.json config
+          file in <Tag intent={Intent.PRIMARY}>blue</Tag>
+          <br />
+          <br />
+          Once your server is correctly setup, we recommend disabling this page by setting the environment variable
+          BP_DISABLE_SERVER_CONFIG to "true"
+        </span>
+      }
+    >
+      {props.children}
+    </PageContainer>
+  )
+}
+
+export const Checklist: FC<Props> = props => {
+  const [langSource, setLangSource] = useState<any>()
+  const [hasAuditTrail, setAuditTrail] = useState(false)
+
+  useEffect(() => {
+    if (!props.serverConfigLoaded) {
+      props.fetchServerConfig()
+    }
+    // tslint:disable-next-line: no-floating-promises
+    loadData()
+  }, [])
+
+  const loadData = async () => {
+    const { data: sources } = await api.getSecured().get('/admin/languages/sources')
+    setLangSource(sources.languageSources)
+
+    const { data: debug } = await api.getSecured().get('/admin/server/debug')
+    const audit = Object.keys(debug)
+      .filter(x => x.startsWith('bp:audit'))
+      .map(x => debug[x])
+
+    setAuditTrail(_.some(audit, Boolean))
+  }
+
+  if (!props.serverConfig) {
+    return (
+      <Container>
+        <Callout intent={Intent.PRIMARY}>
+          Server configuration is disabled. To view this page, set the environment variable "BP_DISABLE_SERVER_CONFIG"
+          to false
+        </Callout>
+      </Container>
+    )
+  }
+
+  const getEnv = (key: string): any => getDisplayValue(_.get(props.serverConfig.env, key))
+  const getConfig = (path: string): any => getDisplayValue(_.get(props.serverConfig.config, path))
+  const getLive = (path: string): any => getDisplayValue(_.get(props.serverConfig.live, path))
+
+  const languageEndpoint = _.get(langSource, '[0].endpoint', '')
```

Comment:
just so you know, using _.get, you don't need the brackets when you query an array.

```js
const langEP = _.get(langSource, '0.endpoint')
```